tests/typecheck/tasks/test_tasks.yaml

```
@@ -0,0 +1,23 @@
+-   case: task
+    main: |
+        from typing_extensions import reveal_type
+        from django.tasks import Task, task
+
+        @task
+        def test_task(x: int) -> int:
+            return x + 1
+
+        reveal_type(test_task) # N: Revealed type is "django.tasks.base.Task[[x: builtins.int], builtins.int]"
+        reveal_type(test_task.call(1)) # N: Revealed type is "builtins.int"
+        reveal_type(test_task.enqueue(1)) # N: Revealed type is "django.tasks.base.TaskResult[[x: builtins.int], builtins.int]"
```

```suggestion
        reveal_type(test_task.enqueue) # N: ...
```

Please, test the signature of the task instead